index.js

```diff
@@ -297,11 +297,6 @@ function rpcType(payload, paramset) {
             break;
         case 'FLOAT':
             val = parseFloat(val);
-            if (val < paramset.MIN) {
```

Comment:
Why do you want to remove that check?  I would like to keep it, it hurts no one :)
My experience with eQ-3 software tells me to do better be pedantic when using their interfaces... ;)
And I'm convinced in most use cases the machine running hm2mqtt.js is much more powerful than the CCU, I would bet that we can't save significant/relevant cpu time when we remove the check.

Reply:
Ok, if the paramset was not loaded this line of code doesn't get executed anyway.

I have in mind that I got an error in some version because the paramset was not populated yet.. (again a Docker-only problem, because you usually start the script without the JSON-"database" that is created on first start).

Reply:
leads me to the idea of shipping a prebuilt paramsetDescription file, would make sense -> #29